android/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 
   <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-      package="com.timozacherl.plugins.CapSiriShortcuts">
```

Comment:
Has this been removed by the automatic migration?

Reply:
Yes, package name has been moved to the build.gradle: https://github.com/lovetodream/capacitor-plugin-siri-shortcuts/pull/58/files/9e2d636f2e98dc2e01ae2aec0ce24ad58ce01bb8#diff-197b190e4a3512994d2cebed8aff5479ff88e136b8cc7a4b148ec9c3945bd65aR21